### PR TITLE
add stopwatches for translator and envoy syncer

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: projects/gloo/pkg/translator/translator.go [hunk 1]
```diff
@@ -85,6 +86,9 @@ func (t *translatorInstance) Translate(
 	defer t.lock.Unlock()
 	ctx, span := trace.StartSpan(params.Ctx, "gloo.translator.Translate")
 	defer span.End()
+	stopwatch := statsutils.NewTranslatorStopWatch("EdgeSnapshotTranslator")
```
Comment (nfuden) [inline projects/gloo/pkg/translator/translator.go:89]: So this is for both translation and validation translation. is that what we intend?

Comment (yuval-k) [reply projects/gloo/pkg/translator/translator.go:89]: I did it this way the for 2 reasons:
- closer approximation for "ux"  time
- the difference between them can hint at time spent in validation 
 
though maybe it is better to just add a metric for validation? and remove the one from translator? WDYT?

**Review by nfuden — APPROVED** [review]

**Review by npolshakova — APPROVED** [review]

**Comment by nfuden** [comment]
/skip-changelog lets just get this in and then build some nicer docs around how to use it as we get some better feedback

**Comment by nfuden** [comment]
/skip-ci previously passed, build bot just flaked on most recent merge from main

**Pull request merged**